addons/project/views/project_views.xml

```diff
@@ -89,6 +89,167 @@
             </field>
         </record>
 
+        <!-- Task types -->
+        <record id="task_type_search" model="ir.ui.view">
```

Comment:
Note: Should adapt code according to the guidelines
```suggestion
        <record id="project_task_type_view_search" model="ir.ui.view">
```

Reply:
@tivisse  This code has just been moved because some references were missing when loading the file for the first time. Shall I keep the name as they were or should I edit all of them (including the ones below ?) => I guess that in that case I should write a migration script accordingly ?

Reply:
Yes, you're right. We can keep it that way then :)